app/Models/User.php

```diff
@@ -487,6 +497,10 @@ public function grantLicense(LicenseType $type): bool
 
     public function revokeLicense(LicenseType $type): bool
     {
+        if ($type == LicenseType::ConversationalAi && Cache::has('ai-users-count') && Feature::active('ai_utilization')) {
+            Cache::forget('ai-users-count');
+        }
```

Comment:
I don't think this is needed anymore since we are just having the caches be cleared and refreshed at a set interval or when a User chooses to refresh the report

Reply:
Done